modules/ossm-security-cert-manage.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_user_guide/ossm-security.adoc
+
+[id="ossm-cert-manage"]
+= Adding an external certificate authority key and certificate
+
+By default, {ProductName} generates self-signed root certificate and key, and uses them to sign the workload certificates. You can also use the operator-specified certificate and key to sign workload certificates, with operator-specified root certificate. This task demonstrates an example to plug certificates and key into {ProductShortName}.
+
+
+
+.Prerequisite
+
+* You must have installed {ProductName} with mutual TLS enabled to configure certificates.
+* Deploy the Bookinfo sample application to verify the results.
+
+== Adding an existing certificate and key
+
+To us an existing signing (CA) certificate and key, you must create a chain of trust file that includes the CA certificate, key, and root certificate. You must use the following exact file names for each of the corresponding certificates. The CA certificate is called `ca-cert.pem`, the key is `ca-key.pem`, and the root certificate, which signs `ca-cert.pem`, is called `root-cert.pem`. If your workload uses intermediate certificates, you must specify them in a `cert-chain.pem` file. 
```

Comment:
Should this be:

To us an existing signing certificate authority (CA) certificate and key,